internal/provider/resource_gitlab_service_jira.go

```diff
@@ -166,46 +166,31 @@ func resourceGitlabServiceJiraCreate(ctx context.Context, d *schema.ResourceData
 
 func resourceGitlabServiceJiraRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(*gitlab.Client)
-	project := d.Get("project").(string)
-
-	p, _, err := client.Projects.GetProject(project, nil, gitlab.WithContext(ctx))
-	if err != nil {
-		if is404(err) {
-			log.Printf("[DEBUG] Removing Gitlab Jira service %s because project %s not found", d.Id(), p.Name)
-			d.SetId("")
-			return nil
-		}
-		return diag.FromErr(err)
-	}
+	project := d.Id()
 
-	log.Printf("[DEBUG] Read Gitlab Jira service %s", d.Id())
+	log.Printf("[DEBUG] Read Gitlab Jira service %s", project)
 
 	jiraService, _, err := client.Services.GetJiraService(project, gitlab.WithContext(ctx))
 	if err != nil {
 		if is404(err) {
-			log.Printf("[DEBUG] gitlab jira service not found %s", project)
+			log.Printf("[DEBUG] gitlab jira service not found %s, removing from state", project)
 			d.SetId("")
 			return nil
 		}
 		return diag.FromErr(err)
 	}
 
-	if v := jiraService.Properties.URL; v != "" {
-		d.Set("url", v)
+	d.Set("project", project)
+	d.Set("url", jiraService.Properties.URL)
+	d.Set("api_url", jiraService.Properties.APIURL)
+	d.Set("username", jiraService.Properties.Username)
+	d.Set("project_key", jiraService.Properties.ProjectKey)
+	// FIXME: The API or go-gitlab doesn't return `jira_issue_transition_id` properly,
+	//        therefore we don't overwrite the value set in the config, in case it's not set in the API
+	//        response. This currently impacts the import of this attribute.
```

Comment:
@timofurrer  - I assume we'll want to create an issue for this FIXME when we merge this so that we can investigate which of these two items it is 😄 .

I have a docker compose (and a jira license) that I can use to test this out, so I can take a look at it later.

Reply:
@PatrickRice-KSC That would be great. 

However, I'm not sure how Jira would be involved here - I think it's a GitLab / go-gitlab issue here only, but please give it a try.
IMHO: we can also wait for that until a merge to hopefully avoid the FIXME entirely ...

Reply:
@PatrickRice-KSC I didn't yet have time to dig into it, but just spotted this fishy part when unmarshalling the JSON from the API and fiddling with that very field: 

https://github.com/xanzy/go-gitlab/blob/65486a91e11bbb63cbf61bce23cfb48ee174b4d5/services.go#L673

Reply:
@timofurrer  Yeah, that definitely seems a bit fishy. I did just try this out, and attaching a JIRA instance doesn't make any difference, it still doesn't unmarshall properly, so we can rule that out.

Reply:
I've created an upstream issue here: https://gitlab.com/gitlab-org/gitlab/-/issues/362437

At the moment I think we can merge this. WDYT? @PatrickRice-KSC

Reply:
Agreed!